src/fabric/NativeScreensModule.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/ban-types */
+// its needed for codegen to work
+import type { TurboModule } from 'react-native';
+import { TurboModuleRegistry } from 'react-native';
+import type { Double, Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+
+export interface Spec extends TurboModule {
```

Comment:
Why is it TurboModule? I believe we don't have currently any TurboModules in screens. Cannot it be just a native module?